### PR TITLE
[STOS-103] Code quality improvement

### PR DESCRIPTION
**Pull request by lukrzak**
Implementation of tasks regarding [[STOS-103]](https://tpalinski.atlassian.net/browse/STOS-103) story

[STOS-103]: https://tpalinski.atlassian.net/browse/STOS-103?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

To use, install dependencies and run command: `pre-commit install`

### CODE REVIEW

**Comment by lukrzak** [comment]
Waiting for merging [[STOS-86]](https://tpalinski.atlassian.net/browse/STOS-86), [[STOS-88]](https://tpalinski.atlassian.net/browse/STOS-88) and [[STOS-89]](https://tpalinski.atlassian.net/browse/STOS-89) before continuing with [[STOS-109]](https://tpalinski.atlassian.net/browse/STOS-109)

[STOS-86]: https://tpalinski.atlassian.net/browse/STOS-86?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[STOS-88]: https://tpalinski.atlassian.net/browse/STOS-88?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[STOS-89]: https://tpalinski.atlassian.net/browse/STOS-89?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[STOS-109]: https://tpalinski.atlassian.net/browse/STOS-109?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Flask~=3.0.3
```
Comment (tpalinski) [inline requirements.txt]: Fajnie by to dac do jakis dev dependencies, żeby nie instalować tego na produkcji

Comment (lukrzak) [reply requirements.txt]: Dodano osobną konfigurację podpisaną jako dev, bo zawiera tylko rzczy związane z lokalnym devem typu walidatory kodu

**Review by tpalinski — APPROVED** [review]
LGTM, gj. Good to merge after squash.

**Comment by tpalinski** [comment]
@lukrzak merge reminder

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# git config --system core.longpaths true
+# To skip, use --no-verify flag in git commit command
```
Comment (tpalinski) [inline .pre-commit-config.yaml:2]: forbidden knowledge

**Review by tpalinski — APPROVED** [review]
Would you like to add prospector to dev nix environment? This way we can integrate the code quality checks into our pre deploy checks. I can also do it in a separate ticket

**Pull request merged**